src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -305,21 +320,47 @@ public function bulkDeleteAction(int $productId, Request $request): JsonResponse
         $combinationIds = $request->request->get('combinationIds');
         if (!$combinationIds) {
             return $this->json([
-                'error' => $this->getFallbackErrorMessage('', 0, 'Missing combinationIds in request body'),
+                'errors' => $this->getFallbackErrorMessage('', 0, 'Missing combinationIds in request body'),
```

Comment:
Not sure this should be changed 🤔. There is only one error message so the singular is more adapted, besides the JS code is able to adapt to both formats